### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by phansys**
|Q            |A   |
|---          |--- |
|Fixed tickets|#611, #614|
|License      |MIT |

Closes #614.

### CODE REVIEW

File: tests/Helper/GitHelperTest.php [hunk 1]
```diff
@@ -102,15 +102,14 @@ public function returns_false_when_top_git_dir_was_expected()
     }
 
     /**
-     * @test
      * @runInSeparateProcess
      */
-    public function returns_false_for_non_git_dir()
+    public function testThrowExceptionForNonGitDir()
```
Comment (GrahamCampbell) [inline tests/Helper/GitHelperTest.php]: why the change in the method casing?

Comment (phansys) [inline tests/Helper/GitHelperTest.php]: This is a different method, the change isn't only about its name.
